### DISCUSSION TITLE
break line promblem

### DISCUSSION THREAD

**Discussion by tom2k3** [SubtitleEdit/subtitleedit#5111] [General]
please fix break line for Chinese subtitle.thanks

https://user-images.githubusercontent.com/28794975/121129658-da02d880-c85f-11eb-9ff6-7fa6eb7edd59.mp4

**Comment by darnn** [comment]
It seems like the problem here is that there's no punctuation at the end of the sentences (which seems like it might be the convention in Chinese subtitles?). In that situation, auto break would work the same in English. However, SE also appears to not recognize the Chinese full stop, the Chinese question mark and so on:
https://en.wikipedia.org/wiki/Chinese_punctuation

So, auto break works with this:
-我很大. -他很小.
But not with this:
-我很大。-他很小。
